src/RefractSourceMap.h

```diff
@@ -37,7 +37,7 @@ namespace drafter {
     }
 
     template<typename T>
-    refract::IElement* LiteralToRefract(const NodeInfo<std::string>& literal)
+    refract::IElement* LiteralToRefract(const NodeInfo<std::string>& literal, ConversionContext& context)
```

Comment:
I don't see it. What I see is that it is called with this parameter not that the parameter is used inside the function, but maybe I am missing something.